.appveyor.yml

```diff
@@ -46,6 +46,8 @@ install:
     - php composer.phar self-update
     - copy /Y .github\composer-config.json %APPDATA%\Composer\config.json
     - php composer.phar global require --no-progress --no-scripts --no-plugins symfony/flex dev-master
+    - git config --global user.email ""
+    - git config --global user.name "Symfony"
```

Comment:
that seems unrelated to the HttpClient test, isn't it ?

Reply:
It's needed for the CI to run successfully, but that's for 3.4 eventually yes.

Reply:
please submit it in a separate PR for 3.4 then.